server/src/internalClusterTest/java/org/opensearch/cluster/metadata/MetadataIndexStateServiceIT.java

```diff
@@ -53,20 +53,24 @@ public void testIndexCloseAndOpen() throws Exception {
                 .get();
         }
 
-        assertAcked(client().admin().indices().prepareClose(TEST_INDEX).get());
-
-        ClusterStateResponse clusterStateResponse = client().admin().cluster().prepareState().get();
-        IndexMetadata indexMetadata = clusterStateResponse.getState().metadata().index(TEST_INDEX);
-        assertEquals(IndexMetadata.State.CLOSE, indexMetadata.getState());
+        CloseIndexResponse closeIndexResponse = client().admin().indices().prepareClose(TEST_INDEX).get();
+        assertBusy(() -> {
```

Comment:
You shouldn't need the assertBusy on a close here, the index should be closed on ack.